### ISSUE TITLE
Thorium can't launch

### ISSUE THREAD

**Issue by Razvy01** [Alex313031/Thorium-Win#82]
System Details
 - OS: Windows 11 Build 22621.2428
 - Thorium Version: 117.0.5938.157

Problem
I'm unable to use this version of Thorium browser, as well as the AVX2 version, even though my CPU is Intel Core I5 10400F, which supports both AVX and AVX2 instruction sets. The only version that's working for me is the one using SS3. Also, it's important to note that when I'm trying to open the browser, the window appears for a brief moment and then disappears.
Any help is highly appreciated!

**Comment by ebintobin** [comment]
I had the same issue, I had to turn off Microsoft Defender to get it working.

**Comment by Razvy01** [comment]
I installed it with Microsoft Defender turned off, but it's still not working. I also tried to get it to work in a VM with Arch, but I got this error:
find: ‘/home/razvy/.config/thorium/Crash Reports/pending/’: No such file or directory
Illegal instruction (core dumped)

**Comment by RUOKexe** [comment]
same problem when i try to instal the browser it said that Illegal instruction error

**Comment by gaberilde** [comment]
+1 to this issue was gonna post my own but found this one so wise not to do that of couse
needs to be looked at i have simalr cpu but mine is i5-11400H

**Comment by RUOKexe** [comment]
I3 10G


‫في السبت، 21 أكتوبر 2023 في 10:55 م تمت كتابة ما يلي بواسطة ‪Gabrielo‬‏ <‪
***@***.***‬‏>:‬

**Comment by vipexv** [comment]
Same issue, Windows 11 Dev Preview.

i5-11400F

**Comment by gz83** [comment]
If your computer has an older processor, it is recommended to use the SSE3 version of the Thorium-SSE3 repo first, and if the processor supports the AVX instruction set or AVX2 instruction set, you can choose the version that corresponds to the instruction set to use.

The Thorium browser may be reported as a virus by some anti-virus programs on Windows systems and may be blocked from launching, this is a false alarm by these anti-virus programs (if you don't trust my reply, you can try to compile it yourselves and you will find that everything is the same as the version we compiled).

**Comment by vipexv** [comment]
I'm still confused, a i5-11400F should support it, and somehow it still doesn't launch, even with Anti virus turned off?

**Comment by vipexv** [comment]
Also one other issue, on any chromium browser somehow it just lags, the animations are laggy and everything else, but when i use firefox or anything else that isn't a chromium based browser it's just fine.

**Comment by gz83** [comment]
> Also one other issue, on any chromium browser somehow it just lags, the animations are laggy and everything else, but when i use firefox or anything else that isn't a chromium based browser it's just fine.

It is recommended to check if there are any abnormalities in your system. Also regarding Thorium: Are you using the mini_installer or the portable version?

**Comment by vipexv** [comment]
Yes i'm using the Mini installer.

**Comment by gz83** [comment]
> Yes i'm using the Mini installer.

Can the browser be installed normally by running mini_installer? Or can't be opened after installation?

@vipexv

**Comment by vipexv** [comment]
It can be installed normally, but it simply doesn't launch, i see a white screen for 1 second and it crashes/closes.

**Comment by gaberilde** [comment]
I have i5-11400H and have the same issue but other apps that require the same thing runs

**Comment by jeswastaken** [comment]
any solutions??

**Comment by gz83** [comment]
Please use the SSE3 version for the time being. I am also curious why these problems occur.

**Comment by jeswastaken** [comment]
can i have a url?

**Comment by jeswastaken** [comment]
> Collaborator

can i have a link?

**Comment by gz83** [comment]
> > Collaborator
> 
> can i have a link?

https://github.com/Alex313031/Thorium-SSE3/releases

@jeswastaken

**Comment by jeswastaken** [comment]
> > > Collaborator
> > 
> > 
> > can i have a link?
> 
> https://github.com/Alex313031/Thorium-SSE3/releases
> 
> @jeswastaken

thanks!

**Comment by amine001100** [comment]
What is the difference between it and normal?

**Comment by bai0012** [comment]
I have also encountered the same issue. Everything was running fine until today, but after I closed Thorium and left my computer for 2 hours, it wouldn't open again. I am using the AVX2 121.0.6167.204 installation version from winget, and I have also tried disabling the protection features of Windows Defender and adding Thorium's folder as an exclusion in Windows Defender. However, Thorium still fails to start up properly.
I hope this information helps you troubleshoot the problem.
